gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -34,6 +34,7 @@ pub async fn run_webserver(
         .route("/connect", post(connect))
         .route("/backup", post(backup))
         .route("/restore", post(restore))
+        .route("/reconnect", post(reconnect))
```

Comment:
`/connect` and `/reconnect` are very confusing and poorly named APIs.
I propose we follow up with a rename like:
- `/connect` > `/connect-fed`
- `/reconnect` > `/connect-ln`

Reply:
#2073